tests/model_serving/model_server/components/kserve_dsc_deployment_mode/conftest.py

```diff
@@ -15,30 +15,58 @@
 from tests.model_serving.model_server.components.kserve_dsc_deployment_mode.utils import (
     patch_dsc_default_deployment_mode,
 )
+from utilities import constants
 from utilities.constants import ModelAndFormat
 from utilities.inference_utils import create_isvc
 
 
+@pytest.fixture(scope="class")
+def inferenceservice_config_cm(admin_client: DynamicClient) -> ConfigMap:
+    return ConfigMap(
+        client=admin_client,
+        name="inferenceservice-config",
+        namespace=py_config["applications_namespace"],
+    )
+
+
 @pytest.fixture(scope="class")
 def default_deployment_mode_in_dsc(
```

Comment:
this fixture needs to be deleted and all code should be under `patched_default_deployment_mode_in_dsc`

`patched_default_deployment_mode_in_dsc` should be renamed to `patched_deployment_service_in_dsc` or alike